### ISSUE TITLE
feat: develop TitleSection component

### ISSUE THREAD

**Issue by salihdev0** [frontendship/website#50]
We developed a component called Title for the application. However, in most of the places we use Title, we also use subtitle. So we can develop a component called TitleSection and expect an optional subtitle.

For example, if we want to use subtitle right now, we use it like this:

```tsx
<div className="flex flex-col items-center">
  <Title level={2}>Frontendship&apos;e neden katılmalıyım?</Title>
  <p className={style.subDescription}>Frontend alanında sıfırdan ileri seviyelere kadar sektöre katkı sağlayacak içerikler üretmek ve topluluğa destek sağlamak istiyoruz.</p>
</div>
```

I think it's better to use it this way:

```tsx
<TitleSection
  title="Frontendship'e neden katılmalıyım?"
  subtitle="Frontend alanında sıfırdan ileri seviyelere kadar sektöre katkı sağlayacak içerikler üretmek ve topluluğa destek sağlamak istiyoruz."
></TitleSection>
```

**Comment by eraygundogmus** [comment]
sounds like a good idea!

**Comment by salihdev0** [comment]
We're gonna do this with @sedaklnc  so I throw it at her

**Comment by sedaklnc** [comment]
you can assign it to me

**Comment by salihdev0** [comment]
merged in #58
